docs/source/network/network.md

```diff
@@ -241,14 +241,7 @@ could have been organization R1 or R4 who created a channel C1. Again, note
 that a channel can have any number of organizations connected to it -- we've
 shown two as it's the simplest configuration.
 
-Again, notice how channel C1 has a completely separate configuration, CC1, to
-the network configuration NC4. CC1 contains the policies that govern the
-rights that R1 and R2 have over the channel C1 -- and as we've seen, R3 and
-R4 have no permissions in this channel. R3 and R4 can only interact with C1 if
-they are added by R1 or R2 to the appropriate policy in the channel
-configuration CC1. An example is defining who can add a new organization to the
-channel. Specifically, note that R4 cannot add itself to the channel C1 -- it
-must, and can only, be authorized by R1 or R2.
+Again, notice how channel C1 has a completely separate configuration, CC1, to the network configuration NC4. CC1 contains the policies that govern the rights that R1 and R2 have over the channel C1. An example of these policies is defining who can add a new organization to the channel. In our example, organizations other than R1 and R2 have no permissions in this channel C1 and can only interact with it if they are added by R1 or R2 to the appropriate policy in the channel configuration CC1. Specifically, note that R4 cannot add itself to the channel C1. It must, and can only, be authorized by R1 or R2.
```

Comment:
What is better. `notice` (Line 244) or `note` (Line 240 above) ?

Reply:
I don't think my edits are applicable to `main` branch. I think most of the paragraphs edited here are already replaced or deleted in the `main` branch.

If you find any of the edits provided here are still relevant, feel free to update `main` branch accordingly.

Reply:
Minor: wrapping lines is much better than having long lines

Reply:
@ale-linux I had actually checked [Style guide for contributors](https://hyperledger-fabric.readthedocs.io/en/release-2.2/style_guide.html) before contributing. It specifically [say](https://hyperledger-fabric.readthedocs.io/en/release-2.2/style_guide.html#formatting-and-punctuation) that:

> Line lengths.
> If you look at the raw versions of the documentation, you will see that many topics conform to a line length of roughly 70 characters. This restriction is no longer necessary, so you are free to make lines as long as you want.

If you still think it would be better to have shorter lines, I would make them shorter, but should the `Style guide for contributors` be updated to reflect this preference too?

Reply:
Long lines are ok now, but I would stick with one sentence per line. It makes it easier to work with especially during edits and reviews.

Reply:
No problem, I will make them shorter.
Can you give an opinion about [this](https://github.com/hyperledger/fabric/pull/3359#discussion_r859720494) ?
> What is better. notice (Line 244) or note (Line 240 above) ?